src/config/app.config.ts

```diff
@@ -1,7 +1,7 @@
 import { registerAs } from '@nestjs/config';
 
 export default registerAs('config', () => ({
-    port: parseInt(process.env.PORT, 10) || 5000,
+    port: parseInt(<string>process.env.PORT, 10) || 5000,
```

Comment:
Use nullish operator instead of assertion

Reply:
Done